File: Entities/Response/CatalogueResponse.cs
namespace Entities.Response
{
    public class CatalogueResponse
    {
        public string Id { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public int State { get; set; }
        public string CreatedAt { get; set; } = string.Empty;
        public string? UpdatedAt { get; set; }
        public string CreatedBy { get; set; } = string.Empty;
        public string? UpdatedBy { get; set; }
    }
}

File: dashboard_react.client/src/types/CatalogueResponse.ts
import { SearchCatalogue } from "./Catalogue";

export interface CatalogueResponse extends SearchCatalogue {
  id: string;
  name: string;
  description: string;
  state: boolean;
  createdAt: string;
  updatedAt?: string;
  createdBy: string;
  updatedBy?: string;
}
